exaudfclient/base/javacontainer/script_options/converter_legacy.cc

```diff
@@ -0,0 +1,26 @@
+#include "base/javacontainer/script_options/converter_legacy.h"
+#include "base/javacontainer/script_options/string_ops.h"
+#include <iostream>
+#include <sstream>
+
+namespace SWIGVMContainers {
+
+namespace JavaScriptOptions {
+
+ConverterLegacy::ConverterLegacy()
+: Converter()
+, m_jarPaths() {}
+
+void ConverterLegacy::convertExternalJar(const std::string& value) {
+    std::istringstream stream(value);
+    std::string jar;
+
+    while (std::getline(stream, jar, ':')) {
+        m_jarPaths.insert(jar);
```

Comment:
The trimming happens in the Parser, or?

Reply:
Yes, [here](https://github.com/exasol/script-languages/blob/302827801e5ee8cff1fda07d9ed8fc010f64a10a/exaudfclient/base/script_options_parser/legacy/script_option_lines.cc#L82)

Also there is unit test ensuring that we don't change the behavior:
https://github.com/exasol/script-languages/blob/302827801e5ee8cff1fda07d9ed8fc010f64a10a/exaudfclient/base/javacontainer/test/cpp/javacontainer_test.cc#L49